src/struct_mv/struct_matmult.c

```diff
@@ -1314,7 +1314,7 @@ hypre_StructMatmultCompute( hypre_StructMatmultData  *mmdata,
    hypre_GpuProfilingPushRange("StructMatmultCompute");
 
    /* Set flag for initializing matrix/vector entries to zeroes */
-#if defined(HYPRE_USING_GPU)
+#if defined(HYPRE_USING_GPU) && defined(HYPRE_FUSE_FCC_FC_F)
```

Comment:
What is the `HYPRE_FUSE_FCC_FC_F` flag? I think this is not defined, so this code path will no longer be taken, correct? I guess the code block here only controls the behavior of `zero_init`, which we disabled for now anyways...